src/main/java/frc/robot/devices/LidarV3.java

```diff
@@ -44,13 +53,15 @@ public void startMeasuring() {
         writeRegister(0x04, 0x08 | 32); // default plus bit 5
```

Comment:
there should probably be some check that start Measuring is only called once as otherwise i think multiple threads would be running. another option would be to make start and stop measuring private